src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings

```diff
@@ -2787,6 +2787,10 @@ prototype_db_prop_encryptionEnabled = Encryption Enabled
 prototype_db_prop_databasescopedconfig_value_off = OFF
 prototype_db_prop_databasescopedconfig_value_on = ON
 prototype_db_prop_databasescopedconfig_value_primary = PRIMARY
+prototype_db_prop_databasescopedconfig_value_when_supported = WHEN_SUPPORTED
```

Comment:
These shouldn't actually be localized, right?

And actually, same for the above (OFF/ON/PRIMARY) - generally we shouldn't be localizing the actual values if they map directly to T-SQL options (and instead rely on descriptions to tell the user what they mean with a localized string if needed)

Reply:
We are localizing the options values in STS and sending them to ADS UI.

@corivera correct me if I'm wrong! thanks

Reply:
I would agree with Charles' opinion here. Were these strings that were already being localized in SSMS?

Reply:
I also took a look at translated versions of the TSQL page for ALTER DATABASE SCOPED CONFIGURATION and the TSQL arguments are not translated either.

Reply:
No, I was looking at the docs page. Here's the page in spanish, for example: https://learn.microsoft.com/es-es/sql/t-sql/statements/alter-database-scoped-configuration-transact-sql?view=sql-server-ver16

You can change the locale ID after the ".com" to see other variations

Reply:
Good to know, Verified the PageVerify and restrictAccess for Spanish and the Page Verify options are not translated there too! Will send a new PR of Un localizing the options.